games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
> I agree on your last paragraph, this is out of the scope of this PR.

Then you should know that this will not be merged with that maintainer removal.